api2-common/src/main/resources/config/annotation.user.properties.template

```diff
@@ -1,6 +1,11 @@
 ### annotation database ###
 # mongoDB settings
 mongodb.annotation.connectionUrl=mongodb://127.0.0.1:27017/annotation_test
+mongodb.annotation.host=localhost
```

Comment:
the code in the develop branch was updated to use only the connection URL, the rest of the properties must be removed. Please merge the develop branch and revert the changes in this file